chat-commands.js

```diff
@@ -1902,6 +1902,51 @@ const commands = {
 			this.errorReply(`User '${target}' is not locked.`);
 		}
 	},
+	unlockname: function (target, room, user) {
+		if (!target) return this.parse('/help unlock');
+		if (!this.can('lock')) return false;
+
+		const userid = toId(target);
+		const punishment = Punishments.userids.get(userid);
+		if (!punishment) return this.errorReply("This name isn't locked");
+		if (punishment[1] === userid) return this.parse(`/unlock ${userid}`);
+
+		Punishments.userids.delete(userid);
+		Punishments.savePunishments();
+
+		Users.findUsers([userid], []).forEach(curUser => {
+			if (curUser.locked && Punishments.getPunishType(curUser.userid) !== 'LOCK') {
```

Comment:
This will unlock a lot of users other than just users locked under a name. For instance, it'll unlock every user on a  VPN.